protocol/avdl/keybase1/ephemeral.avdl

```diff
@@ -16,6 +16,7 @@ protocol ephemeral {
     HashMeta hashMeta;
     EkGeneration generation;
     Time ctime;
+    Time deviceCtime;
```

Comment:
What's the over under on putting `deviceCtime` in the statement, vs storing it just in the local EK store without publishing it?

Reply:
it's only a device specific value so it didn't seem worth it to pipe it up